jobserver/api/jobs.py

```diff
@@ -200,10 +211,22 @@ def handle_alerts_and_notifications(request, job_request, job, github_api):
             user_id=job_request.created_by_id,
         )
 
+
+def handle_job_request_notifications(job_request, github_api):
     if hasattr(job_request, "analysis_request"):
         notify_output_checkers(job_request, github_api)
```

Comment:
Would it be possible to add a test to `test_jobs.py` to check that `notify_output_checkers` is called in the same way that we're checking `send_finished_notification`?